src/hooks/serverStateHooks/usePostCreateComment.ts

```diff
@@ -0,0 +1,19 @@
+import { useMutation } from '@tanstack/react-query';
+
+import { ARTWORK } from '@/constants/API';
+
+import postCreateComment from '../../apis/artwork/postCreateComment';
+
+const usePostCreateComment = (postId: number) => {
```

Comment:
```suggestion
const usePostComment = (postId: number) => {
```

Create가 들어가야 하는 이유가 따로 있을까요? 🤔

Reply:
해당 훅의 동작을 보다 명확하게 설명하는 네이밍으로 짓다보니 Create를 추가하게 되었습니다. 어색하신 부분이 있으시거나 중복되는 느낌이 드시면 Create를 삭제하겠습니다.!

Reply:
Post 자체가 Create와 동일한 의미를 가진다고 생각해서 중복되는 느낌이 없지 않아 있다고 생각합니다! 제거해주시면 감사하겠습니다 :)

Reply:
<img width="444" alt="image" src="https://github.com/user-attachments/assets/a36adb62-f11f-4fad-93ec-8b7f8715b475" />

collection 관련해서도 postCreate로 동일한 네이밍을 활용하는 부분을 확인해서 이 부분도 함께 수정했습니다.!